homeassistant/components/sensibo/select.py

```diff
@@ -90,15 +107,21 @@ def __init__(
     @property
     def current_option(self) -> str | None:
         """Return the current selected option."""
-        return self.entity_description.value_fn(self.device_data)
+        state = self.entity_description.value_fn(self.device_data)
+        if TYPE_CHECKING:
+            assert isinstance(state, str)
+        return state.lower()
```

Comment:
Can we calculate this when we get an updated state, to avoid the transformation each time the property is accessed?

Reply:
Should probably solve this in the upstream package instead so it is lowercase already when the data coordinator get's it so it doesn't need to be handled by HA at all (and can then skip type checking etc).